integrations/aws/session_manager.py

```
@@ -0,0 +1,134 @@
+import os
+from typing import Any
+import aioboto3
+from aws_credentials import AwsCredentials
+from port_ocean.context.ocean import ocean
+from loguru import logger
+
+
+class SessionManager:
+    def __init__(self):
+        self._aws_accessible_accounts: list[dict[str, Any]] = []
+        self._aws_credentials: list[AwsCredentials] = []
+    
+    async def reset(self):
+        application_credentials = await self._get_application_credentials()
+        await application_credentials.update_enabled_regions()
+        self._application_account_id = application_credentials.account_id
+        self._application_session = await application_credentials.create_session()
+
+        self._aws_credentials.append(application_credentials)
+        self._aws_accessible_accounts.append({
+            'Id': self._application_account_id,
+            'Name': 'Current Account'
+        })
+
+        self._organization_reader = await self._get_organization_session()
+        await self._update_available_access_credentials()
+    
+    async def _get_application_credentials(self):
+        aws_access_key_id = ocean.integration_config.get("aws_access_key_id")
+        aws_secret_access_key = ocean.integration_config.get("aws_secret_access_key")
+        if not aws_access_key_id or not aws_secret_access_key:
+            logger.warning("Did not specify AWS access key ID or secret access key, using the role credentials from env.")
```

It should be the same env var naming for bot you can raise an exception